src/modules/chat/actions/types.jsx

```diff
@@ -354,6 +354,40 @@ export const untimeout = {
 	}
 }
 
+export const highlight = {
+	presets: [{
+		appearance: {
+			type: 'icon',
+			icon: 'ffz-i-eye'
+		},
+	}],
+
+	defaults: {},
+
+	required_context: ['user'],
+
+	title: 'Highlight User',
+
+	tooltip(data) {
+		return this.i18n.t('chat.actions.highlight.tooltip', `Highlight ${data.user.login}`);
+	},
+
+	click(event, data) {
+		let key = 'chat.filtering.highlight-temp'
+		let val = this.settings.provider.get(key)
+		if(!val)
+			val = []
+		if(val.includes(data.user.id))
```

Comment:
While you *can* do it this way, I always suggest for performance something like:

```javascript
const idx = val.indexOf(data.user.id);
if ( idx === -1 )
    val.push(data.user.id);
else
    val.splice(idx, 1);
```

Faster, more memory efficient, and just as easy to read.